docs-next/pages/_document.tsx

```diff
@@ -27,7 +27,7 @@ class MyDocument extends Document {
             }}
           />
         </Head>
-        <body>
+        <body className="antialiased">
```

Comment:
moved over from the above file, we probably want this class applied to the body (@JedWatson  to confirm)

Reply:
I'm fine with this 👍 

(I'm also not 100% clear on whether we need this at all, because it may have come over from a previous example, but that's a different issue for a different PR)